### PR TITLE
Move the splash screen to VAS.UI.Gtk2

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: LongoMatch/Main.cs [hunk 1]
```diff
@@ -54,7 +55,7 @@ public static void Main (string[] args)
 			GLib.ExceptionManager.UnhandledException += HandleException;
 			CoreServices.Init ();
 			InitGtk ();
-			var splashScreen = new SplashScreen ();
+			var splashScreen = new SplashScreen (Resources.LoadImage (Constants.SPLASH));
```
Comment (vguzmanp) [inline LongoMatch/Main.cs]: Will this stay here when VAS-3 is finished?

Comment (ylatuya) [inline LongoMatch/Main.cs]: Yes, it's a constant that's only used in the Rift namespace, it's one the examples where it should stay as a constant.

**Comment by vguzmanp** [comment]
One more thing, I get an error compiling the unit tests:

> /Users/vguzman/Development/longomatch/Tests/Integration/TestDatabaseMigrationV0.cs(67,67): Error CS0246: The type or namespace name `IProgressReport' could not be found. Are you missing `VAS.Core.Interfaces' using directive? (CS0246) (Tests)

**Comment by vguzmanp** [comment]
Adding the missing using fixes it

**Pull request merged**